docs/contracts/v3/reference/deployments/Optimism Deployments.md

```diff
@@ -0,0 +1,62 @@
+---
+id: optimism-deployments
+title: Optimism Deployments
+---
+
+# Uniswap Contract Deployments
+
+The latest version of `@uniswap/v3-core`, `@uniswap/v3-periphery`, and `@uniswap/swap-router-contracts` are deployed at the addresses listed below. Integrators should **no longer assume that they are deployed to the same addresses across chains** and be extremely careful to confirm mappings below.
+
+| Contract                                                                                                                                                     | Optimism Addresses                           | OP Sepolia Addresses  SDK-Core               | OP Sepolia Addresses                         |
```

Comment:
we discussed in slack but i think maybe we should only have one set of contracts for OP sepolia in the docs

Reply:
I just realized that @aadams deployed to **OP Sepolia Addresses** (see https://uniswapteam.slack.com/archives/C03BSGBKWA0/p1706204854471729?thread_ts=1706204849.006929&cid=C03BSGBKWA0), meanwhile @sbvegan deployed to **OP Sepolia Addressed SDK-Core** (see https://github.com/Uniswap/deploy-v3/issues/20).

But... I think the corresponding universal router deploy (see https://uniswapteam.slack.com/archives/C03BSGBKWA0/p1706204888464989?thread_ts=1706204849.006929&cid=C03BSGBKWA0) happened right before v2-deploy-everywhere, so likely on OP Sepolia, we don't have v2 routing support yet.

I'm a bit leaning toward calling  **OP Sepolia Addresses** (see https://uniswapteam.slack.com/archives/C03BSGBKWA0/p1706204854471729?thread_ts=1706204849.006929&cid=C03BSGBKWA0) a canonical set of deployments, and update the SDK, but  would also respect the process, if foundation gets to vote for deploy addresses on testnets as well.

Reply:
I would not mind shifting to the ones Austin launched. I'm pretty sure there is no v2 testnets outside Goerli fully functional or in the UI (except bad Sepolia), and we need to finish the changes for Sepolia. On the other chains it is still only v3 Sepolia. The [PR mentioned here](https://github.com/Uniswap/docs/pull/683#discussion_r1540145750) was to finish the v2 sepolia testnet.

Maybe we can make a big push to update for v2 and v3 testnets (currently only v3 Goerli rn) and switch to Austin's version. I can leave that version instead here. (it is already in the docs). 

ETH Sepolia should be good for v2 https://github.com/Uniswap/universal-router/blob/main/deploy-addresses/sepolia.json routing, just need the v2router02 [updated in SDK](https://github.com/Uniswap/sdk-core/pull/131)

Making updates for other changes will keep both for now. I think having both in there will let devs know if/why they have had issues. We can rename the column to SDK changing to these OP Sepolia contracts. We will have to update if/when we launch v2 testnets everywhere.
We can speak more on pros vs cons before publishing